bin/run-standard-tests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -eo pipefail
+
+display_and_run() {
+    echo "***" "$@"
+	eval $(printf '%q ' "$@")
+}
+
+# reset workdir to state from git (to remove possible rewritten dependencies)
+display_and_run git reset --hard
+
+# Fmt check
+echo "*** go fmt ./..."
+find . -name '*.go' -not -path "./.gopath/*" -not -path "./vendor/*" -exec gofmt -l {} + > go-fmt.out
+if [[ -s go-fmt.out ]]; then
+	echo "ERROR: some files not gofmt'ed:"
+	cat go-fmt.out
+	exit 1
+fi
+
+
+# Environment
+export GO111MODULE=on
+
+
+list_buildable() {
+    go list -f '{{if (len .GoFiles)}}{{.ImportPath}} {{if .Module}}{{.Module.Dir}}{{else}}{{.Dir}}{{end}}{{end}}' ./... | grep -v /vendor/\|/.gopath/
+}
+
+build_all() {
+# Make sure everything can compile since some package may not have tests
+# Note: that "go build ./..." will fail if some packages have only
+#   tests (will get "no buildable Go source files" error) so we
+#   have to do this the hard way.
+list_buildable | while read -r pkg dir; do
+    echo '*** go build' "$pkg"
+    buildmode=archive
+    if [[ "$(go list -f '{{.Name}}')" == "main" ]]; then
+        # plugin works even when a "main" function is missing.
+        buildmode=plugin
+    fi
+    ( cd "$dir"; go build -buildmode=$buildmode -o /dev/null "$pkg")
+done
+}
+
+list_testable() {
+go list -f '{{if or (len .TestGoFiles) (len .XTestGoFiles)}}{{.ImportPath}}{{end}}' ./... | grep -v /vendor/ || true
+}
+
+# build all packages in the repo
+build_all
+list_testable > packages-with-tests
```

Comment:
:hairy_eyebrow:

Still don't understand why this script is in such denial about `./...`.

Reply:
again trying to exclude certain directories I think, and neccesary here because we do not want to run against .gopath